package.json

```diff
@@ -40,8 +40,8 @@
     "test": "run-s format:check test:types db:clean db:run test:run db:clean && node test/smoke.cjs && node test/smoke.mjs",
     "test:run": "jest --runInBand --coverage",
     "test:update": "run-s db:clean db:run db:generate-test-types && jest --runInBand --updateSnapshot && run-s db:clean",
-    "test:types": "run-s build && tsd --files 'test/**/*.test-d.ts'",
-    "test:types:watch": "run-s build && tsd --files 'test/**/*.test-d.ts' --watch",
+    "test:types": "run-s build && tsd --files 'test/**/*.test*.ts'",
+    "test:types:watch": "chokidar 'src/select-query-parser/**/*.ts' 'test/**/*.ts' -c 'npm run test:types'",
```

Comment:
Previous watcher weren't working. Use chodikar for better DX to check the types errors.